app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/learneranalytics/ProfileAndDeviceIdFragmentTest.kt

```diff
@@ -162,6 +153,8 @@ class ProfileAndDeviceIdFragmentTest {
 
   @Before
   fun setUp() {
+    TestPlatformParameterModule.forceEnableEditAccountsOptionsUi(true)
```

Comment:
Does this one need to be enabled for this test suite? It seems unrelated.

Reply:
removed in last commit